_features/discoverable.html

```diff
@@ -2,34 +2,21 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities. As a developer, I want
+  my web apps to be promoted in search engines.</p>
+- <p>As a search engine owner, I want to highlight those websites identifiable as applications,
+  promoting them in the search results.</p>
+- <p>As a browser developer, I want to give apps special capabilities and allow
+  them to <em>escape the tab</em>.</p>
+- <p>As an app store owner, I want to gather information about progressive web
+  apps to rate and organize them.</p>
 technologies:
 - web-manifest
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
-<!--
-<h2>Use cases</h2>
-<ul class="use-cases">
-  <li>As a user, I want to revisit a site and get its contents even if no network is available.</li>
-  <li>As a user, I want to browse any kind of content I visited at least once, even under situations of poor connectivity.</li>
-  <li>As a developer, in situations with no connectivity, I want to control what is shown to the user.</li>
-</ul>
-<h2>Technologies</h2>
-<ul class="grid">
-  <li>
-    <a href="../technologies/service-workers.html">
-      <h3 id="Service Workers">Service Workers</h3>
-      <p>Cache network resources and enable <strong>super fast</strong> access to them.</p>
-    </a>
-  </li>
-  <li>
-    <h3 id="App Shell">App Shell</h3>
-    <p>Use app shell architecture to reach almost instant first draw.</p>
-  </li>
-</ul>
-<h2>Resources</h2>
-<ul class="resources">
-  <li><a href="https://github.com/GoogleChrome/sw-precache" target="_blank">sw-precache</a> is a node module to generate service worker code that will precache specific resources.</li>
-  <li><a href="https://github.com/mozilla/oghliner" target="_blank">oghliner</a> is not only a template but a tool for deploying Offline Web Apps to GitHub Pages.</li>
-  <li><a href="https://www.talater.com/upup/" target="_blank">UpUp</a>is a tiny script that makes sure your site is always there for your users.</li>
-</ul>
--->
+<p>As soon as you start writing progressive web apps, you want the world to
+  know about them. If a user visits your app in a modern browser, all your
+  web site's progressive super-powers simply emerge but, <em>what if the user
```

Comment:
The comma is still out of place: `but,` -> `,but`

Also, "web site" is spelled as two words here, while elsewhere it's spelled as one "website." Either is fine, but the document should be consistent.